ecl/hql/hqlfold.cpp

```diff
@@ -6540,7 +6552,10 @@ IHqlExpression * CExprFolderTransformer::createTransformed(IHqlExpression * expr
 
 #ifdef LOG_ALL_FOLDING
     if ((op != transformed->getOperator()) || (expr->numChildren() != transformed->numChildren()))
-        DBGLOG("Folding %s to %s", getOpString(updated->getOperator()), getOpString(transformed->getOperator()));
+    {
+        if (doTrace(traceOptimizations))
```

Comment:
minor: Can doTrace wrap the if block two lines above?

Reply:
It could, but it is inside a #ifdef that is not defined.